app/README.md

```diff
@@ -82,37 +76,176 @@ Then add it to the Switch:
 
 Forms can be very frustrating and and difficult to manage, so here is an attempt to find some standardization in the form creation process. Feel free to add and improve this process.
 
+Our forms use three main technologies:
+- [`Formik`](https://jaredpalmer.com/formik/) for form state management
+- [`Yup`](https://github.com/jquense/yup) for validation schemas
+- [`Material UI`](https://material-ui.com) for UI components
+
 ### File Structure
 
-In the hopes of having single-purpose components, we have a separation of the actual form html element (plus select methods), and the logic/validation/state. The forms are utilizing the [`Formik`](https://jaredpalmer.com/formik/) library, [`Yup`](https://github.com/jquense/yup) library, and the [`Material UI`](https://material-ui.com) library. This choice was largely made because creating a form, valiation, & UI system could take a few Hack Oregon seasons and we are volunteers with a deadline 😅. So let's dig a little deeper into how the forms are set up. Ultimately, there are 2 components: the form and the form logic/validation/state.
+There are a few main goals with our form structure:
+- separation of concerns: ui components, form state and validation, application state.
+- create patterns that are easy to reason about, improve, and maintain
+
+This choice was largely made because creating a form, valiation, & UI system could take a few Hack Oregon seasons and we are volunteers with a deadline 😅. So let's dig a little deeper into how the forms are set up.
+
+The example below shows the structure for the ChangePassword form:
+```
+/components
+--- /Fields
+--- PasswordField.js <- Common field UI component
+
+--- /Forms
+--- ChangePasswordForm.js <- Specific form component (validation, fields)
+--- Form.js <- Common Form component used in all Specific form components
+
+
+/page
```

Comment:
Hey @jaronheard, did we talk moving more of the page stuff into the components like this?  

/components/Forms/ChangePassword/
  | ChangePasswordForm.js
  | ChangePassword.js
  | index.js

Reply:
Yes, going to address this in #341